MdePkg/Library/UefiPciSegmentLibPciRootBridgeIo/PciSegmentLib.c

```diff
@@ -184,7 +184,13 @@ DxePciSegmentLibPciRootBridgeIoReadWorker (
   EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL  *PciRootBridgeIo;
 
   PciRootBridgeIo = PciSegmentLibSearchForRootBridge (Address);
-  ASSERT (PciRootBridgeIo != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (PciRootBridgeIo != NULL) {

```

Comment:
this logic seems inverted.
```suggestion
  if (PciRootBridgeIo == NULL) {
```